### PR TITLE
Update documents

### PR DESCRIPTION
**Pull request by cosmiccoincidence**
Fixes broken link on Contributing.md

Also updates Readme.md now with a twitter icon, new download link, and re-organizes the page. You can preview how it looks on my branch: 

https://github.com/cosmiccoincidence/SS3D/tree/link

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,33 +2,53 @@
...
 
-Our YouTube is where we upload many of our important videos including 3D animations, trailers, and gameplay clips of teasers/tests.
+We mostly use our twitter for annoucements and memes currently so definitely go follow us if that's your'e thing.
 
 ## BOARDS:
 
 [<img src="Documents/Images/github.png" alt="github" width="150" align="left">](https://github.com/RE-SS3D/SS3D/projects)
 
-This GitHub repo hosts our project board for tracking the completion status of the game itself, via linking GitHub issues and pull requests.
+This GitHub repository hosts our [issues page](https://github.com/RE-SS3D/SS3D/issues) and [wiki page](https://github.com/RE-SS3D/SS3D/wiki). For more info on our issues you can check the [project boards](https://github.com/RE-SS3D/SS3D/projects) and [milestones pages](https://github.com/RE-SS3D/SS3D/milestones).
```
Comment (Seteron) [inline README.md:51]:
```suggestion
This GitHub repository hosts our [issues page](https://github.com/RE-SS3D/SS3D/issues) and [wiki page](https://github.com/RE-SS3D/SS3D/wiki). For more info on our issues you can check the [project boards](https://github.com/RE-SS3D/SS3D/projects) and [milestone pages](https://github.com/RE-SS3D/SS3D/milestones).
```
Minor grammar nitpick

File: README.md [hunk 2]
```diff
@@ -50,20 +70,4 @@ Subscribestar is our other subscription-based donation platform. Has fewer featu
 
 [<img src="Documents/Images/crypto.png" alt="Crypto" width="150" align="left">](SpaceStation3D.crypto)
```
Comment (Seteron) [inline README.md:71]: This link doesn't work. Redirects to github 404.

**Review by Seteron — CHANGES_REQUESTED** [review]
I noticed a few spelling errors.

File: README.md (hunk 2 above)
Comment (cosmiccoincidence) [reply README.md:71]: 'SpaceStation3d.crypto' doesn't work as a real link unless we host the domain, which isn't necessary for us. We use it as a redirect for the 4 cryptos we accept so people don't need to send directly to a hash address (and it doesn't need to be hosted for this).

I will just remove the link part of this to reduce confusion.

File: README.md [hunk 3]
```diff
@@ -42,13 +42,13 @@ We also have a subreddit, where you will find similar discussion threads and pos
 
 [<img src="Documents/Images/Twitter.png" alt="Twitter" width="150" align="left">](https://twitter.com/SpaceStation3D)
 
-We mostly use our twitter for annoucements and memes currently so definitely go follow us if that's your'e thing.
+Currently our twitter is mostly used for announcements & memes but we will start using it more for the people that prefer it over reddit or discord.
```
Comment (Seteron) [inline README.md:45]: This still looks weird to me but it's good enough to PR.

**Review by Seteron — APPROVED** [review]
LGTM.

**Pull request merged**